### DISCUSSION TITLE
[Core Protocol] eth_sendRawTransactionConditional

### DISCUSSION THREAD

**Discussion by algotradingstrategies** [ethereum-optimism/developers#639] [Application Development]
### Did you check the documentation?

- [X] Yes
- [ ] No

### What type of issue are you experiencing?

- [ ] Error
- [ ] Feature Request
- [X] General Question

### Issue Description

I was trying to send transactions to the sequencer (https://mainnet-sequencer.optimism.io) using  eth_sendRawTransactionConditional , but am getting the error "only 4337 Entrypoint contract support" . 
Is the endpoint eth_sendRawTransactionConditional really only related to AA? I was hoping I can use it to send transactions with a max BlockNumber to include, or is that not possible?

### Steps to Reproduce

    const tx = {
      nonce: nonce,
      to: ethers.constants.AddressZero,
      data: '0x',
      gasPrice: gasPrice,
      gasLimit: gasLimit,
      chainId : 10
    }           
    const signed_tx = await connectedWallet.signTransaction(tx)
    const response = await axios.post(rpcUrl, {
      jsonrpc: '2.0',
      method: 'eth_sendRawTransactionConditional',
      params: [signed_tx,             {
                blockNumberMax: blockNumberMax,
              }],
      id: 1,

### Are you using a specific library or SDK?

- [ ] OP-SDK
- [ ] Viem
- [ ] Other
- [ ] Not Sure

### Are you using the latest version of the library/SDK?

- [ ] Yes
- [ ] No

### What type of support do you need?

- [ ] Technical support for issue resolution
- [ ] Feature implementation guidance
- [ ] General advice or consultation
- [ ] Documentation or learning resources

**Comment by imtipi** [comment]
this method is for AA wallet only
see: https://notes.ethereum.org/@yoav/SkaX2lS9j

**Reply by algotradingstrategies** [reply]
unfortunately, not very familiar with AA yet.

as I wrote, my interest is to send transactions with a max BlockNumber to include. is there any way to "wrap" the tx I want to send into AA wallet world, and send it with eth_sendRawTransactionConditional and a maxBlockNumber?
